### DISCUSSION TITLE
Bitorder for `RgbX`?

### DISCUSSION THREAD

**Discussion by TornaxO7** [image-rs/image#2055] [Q&A] [answered]
Hello! May I ask what the Bitorder (LSB or MSB) has to be for [ColorType](file:///home/tornax/Programming/flakeshot/target/doc/image/enum.ColorType.html)::RgbX (Rgb8, Rgb16 and Rgb32F)?

**Comment by fintelia** [comment, marked as answer]
They're all native endian with the red channel first, followed by green, followed by blue.
